src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -669,4 +670,35 @@ public static boolean isInt(String str) {
         return isInt;
     }
 
+    /**
+     * Counts empty lines before given.
+     * @param lineNo
+     *        Line number of the given line, one-based.
+     * @param lines
+     *        The given lines.
+     * @return count of empty lines before given.
+     * @throws IllegalArgumentException when lines is null or empty, or lineNo is out of range
+     */
+    public static int getCountOfEmptyLinesBefore(int lineNo, String... lines) {
```

Comment:
As I raised this before.

this method should be moved back to Check and covered by UTs based on Input files, 
It is not NOT ok to cover it by pure UTs. We do testing by pure UTs (on fake file content) only when it is impossible to cover it Inputs, and there should be good reason for it.

@pbludov , please pay attention to my comment.

Even more it look strange as common util ... it is not a util method, it is specific for certain Check.

Reply:
Refactored and keep `getCountOfEmptyLinesBefore()` in `CustomImportOrder`